runner/types.go

```diff
@@ -0,0 +1,72 @@
+package runner
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/projectdiscovery/cryptoutil"
+	"github.com/projectdiscovery/httpx/common/httpx"
+)
+
+// List of all dsl variables
+var dslVariables = []string{"timestamp", "csp", "domains", "hash", "body_md5", "body_mmh3", "body_sha256", "body_simhash", "header_md5", "header_mmh3", "header_sha256", "header_simhash", "port", "url", "input", "location",  "scheme", "webserver", "content-type", "method", "host", "path", "a", "words", "lines", "status_code", "content_length", "failed", "time", "body", "header", "request"}
```

Comment:
We need to obtain this information dynamically or have tests to ensure new json keys are added and are also available as DSL variables.

Reply:
we can use reflection or this https://github.com/fatih/structs package which can be used to get json tags

Reply:
used reflect to acess DSL variables dynamically